examples/example-protocol/src/assets/rust_wasmer_runtime_test/expected_bindings.rs

```diff
@@ -24,17 +24,9 @@ impl Runtime {
         Ok(Self { module })
     }
 
-    #[cfg(any(target_arch = "arm", target_arch = "aarch64"))]
     fn default_store() -> wasmer::Store {
-        let compiler = wasmer_compiler_cranelift::Cranelift::default();
-        let engine = wasmer_engine_universal::Universal::new(compiler).engine();
-        Store::new(&engine)
-    }
-
-    #[cfg(not(any(target_arch = "arm", target_arch = "aarch64")))]
-    fn default_store() -> wasmer::Store {
-        let compiler = wasmer_compiler_singlepass::Singlepass::default();
-        let engine = wasmer_engine_universal::Universal::new(compiler).engine();
+        let compiler = wasmer::Cranelift::default();
```

Comment:
I highly suspect this change would already fix the tests:

```suggestion
        let compiler = wasmer::Singlepass::default();
```